java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S2057_java.html

```diff
@@ -31,10 +31,12 @@ <h2>Compliant Solution</h2>
 }
 </pre>
 <h2>Exceptions</h2>
-<p>Swing and AWT classes, <code>abstract</code> classes, <code>Throwable</code> and its subclasses (<code>Exception</code>s and <code>Error</code>s),
-and classes marked with <code>@SuppressWarnings("serial")</code> are ignored.</p>
+<p>Records, Swing and AWT classes, <code>abstract</code> classes, <code>Throwable</code> and its subclasses (<code>Exception</code>s and
+<code>Error</code>s), and classes marked with <code>@SuppressWarnings("serial")</code> are ignored.</p>
 <h2>See</h2>
 <ul>
   <li> <a href="https://wiki.sei.cmu.edu/confluence/x/ajdGBQ">CERT, SER00-J.</a> - Enable serialization compatibility during class evolution </li>
+  <li> <a href="https://docs.oracle.com/en/java/javase/15/docs/specs/records-serialization.html#serialization-of-records">Record Serialization</a> -
```

Comment:
I updated the link in a PR (https://github.com/SonarSource/rspec/pull/169) to link java 16 documentation. Could you update?